posthog/temporal/batch_exports/spmc.py

```diff
@@ -443,6 +443,9 @@ def consumer_done_callback(task: asyncio.Task):
         nonlocal consumer_tasks_done
         nonlocal consumer_tasks_pending
 
+        if task.cancelled():
+            consumer.logger.debug("Record batch consumer task cancelled")
+
```

Comment:
I think we could move this to the except clause below: `result` will raise a `CancelledError` if the task was cancelled.

Reply:
A bit nitpicky though, not critical.

Reply:
yeah I thought it might too but when I tested it didn't seem to, which is why I tested for cancelled directly